### DISCUSSION TITLE
Review Code task 1

### DISCUSSION THREAD

**Discussion by WrElisey** [pavelhabrusevich/AmastyTraining#3] [General]
Видеоревью: https://www.dropbox.com/s/18pbhcneczcui0e/pavelhabrusevich-task1.mkv?dl=0

**Comment by pavelhabrusevich** [comment]
Спасибо.
Согласен со всем.
Наверное в описании поправить надо
![image](https://user-images.githubusercontent.com/33491797/111152116-6f643880-85a1-11eb-8925-c0a62f3f79fb.png)
